### PR TITLE
iterating through concept file.

### PR DESCRIPTION
**Pull request by richardspatrick97**

### CODE REVIEW

**Review by jonaromero86 — COMMENTED** [review]
Code looks good, but I believe there's some variables missing to use the assert method.

**Comment by jonaromero86** [comment]
Jus added method assertConcept(). Left some validations commented.
Please review and help me out on this. Thanks.

File: snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/ConceptSemanticIT.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+//                            "\t" + (descriptionStatus.equals(StateSet.ACTIVE) ? "Active" : "Inactive") +
+//                            "\t" + descriptionType.description() +
+//                            "\t" + caseSensitivityConcept.description() + "\n");
+//                }
+
+            }
+        }
+    }
+
+    private boolean assertConcept(UUID id, String term, EntityProxy.Concept nameType, EntityProxy.Concept caseSensitive, long effectiveDate, StateSet activeFlag) {
```
Comment (stenkarl) [inline snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/ConceptSemanticIT.java]: For asserting a concept, there won't be any "term", "nameType" or "caseSensitive" parameters. All you'll want to do is create the StampCalculator, look it up with the EntityService with the passed in id and then assert that there's a "latest" version of it with the stamp calculator.

File: snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/ConceptSemanticIT.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            while ((line = br.readLine()) != null) {
+                if (line.startsWith("id")) continue;
+                String[] columns = line.split("\\t");
+
+                //pass these args in assertion method
+                UUID id = UuidUtil.fromSNOMED(columns[0]);
+                long effectiveTime = SnomedUtility.snomedTimestampToEpochSeconds(columns[1]);
+                StateSet descriptionStatus = Integer.parseInt(columns[2]) == 1 ? StateSet.ACTIVE : StateSet.INACTIVE;
+                EntityProxy.Concept moduleId = EntityProxy.Concept.make(PublicIds.of(UuidUtil.fromSNOMED(columns[3])));
+                EntityProxy.Concept definitionStatus = EntityProxy.Concept.make(PublicIds.of(UuidUtil.fromSNOMED(columns[4])));
```
Comment (stenkarl) [inline snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/ConceptSemanticIT.java]: I don't think we use the definition status anywhere when we create the concept.

File: snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/ConceptSemanticIT.java (hunk 1 above)
Comment (richardspatrick97) [reply snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/ConceptSemanticIT.java]: The concept file doesnt have a term, case sensitivity, or nametype.

File: snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/ConceptSemanticIT.java (hunk 2 above)
Comment (richardspatrick97) [reply snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/ConceptSemanticIT.java]: looking further into it, I see that we dont, you are correct, will delete.

**Review by stenkarl — APPROVED** [review]

**Pull request merged**